src/components/Layout/Layout.tsx

```diff
@@ -21,21 +21,20 @@ import {
   DROPDOWN_MENU_SIGN_OUT_EVENT,
 } from 'decentraland-dapps/dist/containers/Navbar/constants'
 import useNotifications from 'decentraland-dapps/dist/hooks/useNotifications'
+import { shorten } from 'decentraland-ui/dist/components/AddressField/utils'
 import {
   Footer,
   FooterProps,
 } from 'decentraland-ui/dist/components/Footer/Footer'
 import { Locale } from 'decentraland-ui/dist/components/LanguageIcon/LanguageIcon'
-import { config } from 'decentraland-ui2/dist/config'
-
+import { Navbar } from 'decentraland-ui/dist/components/Navbar/Navbar'
+import { NavbarProps } from 'decentraland-ui/dist/components/Navbar/Navbar.types'
```

Comment:
```suggestion
import { Navbar, NavbarProps } from 'decentraland-ui/dist/components/Navbar'
```

Reply:
in UI1 we are not exporting all those in a `index.ts`